package.json

```diff
@@ -12,6 +12,7 @@
   ],
   "scripts": {
     "bundle": "webpack",
+    "bundle:prod": "NODE_ENV=production webpack -p",
```

Comment:
Is it necessary to specify `NODE_ENV=production`? Shouldn't the `-p` flag already handle that?